view/frontend/web/js/view/payment/method-renderer/lpm.js

```diff
@@ -0,0 +1,260 @@
+                    region: shippingAddress.region,
+                    countryCode: shippingAddress.countryId
+                }
+            },
+
+            getAmount: function () {
+                return quote.totals()['base_grand_total'].toString();
+            },
+
+            getBillingAddress: function () {
+                return quote.billingAddress();
+            },
+
+            getClientToken: function () {
+                return window.checkoutConfig.payment[this.getCode()].clientToken;
+            },
+
+            getCode: function () {
+                return this.code;
+            },
+
+            getCurrencyCode: function () {
+                return quote.totals()['base_currency_code'];
+            },
+
+            getCustomerDetails: function() {
+                var billingAddress = quote.billingAddress();
+                return {
+                    firstName: billingAddress.firstname,
+                    lastName: billingAddress.lastname,
+                    phone: billingAddress.telephone,
+                    email: typeof quote.guestEmail === 'string' ? quote.guestEmail : billingAddress.email
+                }
+            },
+
+            getData: function () {
+                let data = {
+                    'method': this.getCode(),
+                    'additional_data': {
+                        'payment_method_nonce': this.paymentMethodNonce,
+                    }
+                };
+
+                data['additional_data'] = _.extend(data['additional_data'], this.additionalData);
+
+                return data;
+            },
+
+            getMerchantId: function () {
+                return window.checkoutConfig.payment[this.getCode()].merchantId;
+            },
+
+            getPaymentMethod: function(method) {
+                var methods = this.getPaymentMethods();
+
+                for (var i = 0; i < methods.length; i++) {
+                    if (methods[i].method === method) {
+                        return methods[i]
+                    }
+                }
+            },
+
+            getPaymentMethods: function() {
+                return window.checkoutConfig.payment[this.getCode()].allowedMethods;
+            },
+
+            getPaymentMarkSrc: function () {
+                return window.checkoutConfig.payment[this.getCode()].paymentIcons;
+            },
+
+            getTitle: function() {
+                return window.checkoutConfig.payment[this.getCode()].title;
+            },
+
+            initialize: function () {
+                this._super();
+
+                var self = this;
+
+                return this;
+            },
+
+            isActive: function() {
+                var address;
+
+                if (!quote.billingAddress()) {
+                    address = quote.shippingAddress();
+                } else {
+                    address = quote.billingAddress();
+                }
```

Comment:
I think this would do the same thing
`var address = quote.billingAddress() || quote.shippingAddress();`